testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -566,3 +566,18 @@ def test_alignto_reorder_atomgroups():
     ref = u.atoms[[3, 2, 1, 0]]
     rmsd = align.alignto(mobile, ref, select='bynum 1-4')
     assert_allclose(rmsd, (0.0, 0.0))
+
```

Comment:
Please see do address the various issues pep8speaks has identified: https://github.com/MDAnalysis/mdanalysis/pull/3562#issuecomment-1069411688

Here you would want 2 empty lines instead of one.
```suggestion


```